projects/C/atof.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+#include <ctype.h>
+
+/*convert string s to double*/
+double atof(char s[])
+{
+	int i, sign;
+	double val, power;
+	for (i = 0; isspace(s[i]); ++i)
+		;
```

Comment:
https://google.github.io/styleguide/cppguide.html#Loops_and_Switch_Statements

```suggestion
	for (i = 0; isspace(s[i]); ++i)
		continue;
```